NEWS

```diff
@@ -11,9 +11,143 @@
 ### Features:
 ### Bugfixes:
 
+## 1.15.0-rc2 (July 27, 2023)
+### Features:
+#### RDMA CORE (IB, ROCE, etc.)
+* Implemented is_reachable_v2 for IB interfaces
+#### Build
+* Enabled build with binutils 2.40
+* Removed inbox packages on install
+
+### Bugfixes:
+#### UCP
+* Reverted the change in lanes selection to fix broken wire compatibility
+* Fixed endpoint reconfiguration error due to wrong endpoint configuration locality assignment
+#### RDMA CORE (IB, ROCE, etc.)
+* Fixed performance degradation when KSM with atomic access fails repeatedly during rkey_pack phase of rendezvous protocol
```

Comment:
too much details. Nobody knows what is KSM

Reply:
Removed details.